File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        Map<Integer, Integer> storeCount = new HashMap<>();
        if(nums.length == 0) return false;
        if(nums.length == 1) return false;

        for (int j : nums) {
            if (storeCount.containsKey(j)) {
                storeCount.put(j, storeCount.get(j) + 1);
            } else {
                storeCount.put(j, 1);
            }
        }
        for(Map.Entry<Integer, Integer> entry : storeCount.entrySet()) {
            if(entry.getValue() > 1) {
                return true;
            }
        }
        return false;
    }
}

File: Data Structures & Algorithms/concatenation-of-array/submission-0.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        ans = [0]*(len(nums)*2)
        i = 0
        j = 0
        while i < len(ans):
            if j == len(nums):
                j = 0
            print(i, j)
            ans[i] = nums[j]
            i+=1
            j+=1
        return ans